dependents-data/semistandard.json

```diff
@@ -0,0 +1,8 @@
+[
+  "attentif/unix-timestamp-js",
+  "erdtman/COSE-JS",
+  "feathersjs-ecosystem/feathers-swagger",
+  "feathersjs-ecosystem/feathers-sync",
+  "lovell/icc",
+  "onigoetz/postcss-lightningcss"
+]
```

Comment:
This list used to be generated on the fly in [.github/workflows/compatibility.yml](https://github.com/neostandard/neostandard/pull/121/files#diff-0ff6bcc45c7c09bfcb5419c83559f190add597ec3b82ad238ddb1d6e43e9517e) but is now generated on updates through the `dependents-filter-simplify` npm script instead